geospaas_harvesting/ingesters.py

```diff
@@ -539,15 +540,30 @@ def _get_normalized_attributes(self, dataset_info, *args, **kwargs):
         # set optional ForeignKey metadata from Nansat or from defaults
         normalized_attributes['gcmd_location'] = n_metadata.get(
             'gcmd_location', pti.get_gcmd_location('SEA SURFACE'))
-        normalized_attributes['provider'] = n_metadata.get(
-            'data_center', pti.get_gcmd_provider('NERSC'))
+        normalized_attributes['provider'] = pti.get_gcmd_provider(
+            n_metadata.get('provider', 'NERSC'))
         normalized_attributes['iso_topic_category'] = n_metadata.get(
             'ISO_topic_category', pti.get_iso19115_topic_category('Oceans'))
 
         # Find coverage to set number of points in the geolocation
         if len(nansat_object.vrt.dataset.GetGCPs()) > 0:
             nansat_object.reproject_gcps()
         normalized_attributes['location_geometry'] = GEOSGeometry(
-            nansat_object.get_border_wkt(nPoints=n_points), srid=4326)
+            nansat_object.get_border_wkt(n_points=n_points), srid=4326)
+        json_dumped_dataset_parameters = n_metadata.get('dataset_parameters', None)
+        if json_dumped_dataset_parameters:
+            try:
+                json_loads_result = json.loads(json_dumped_dataset_parameters)
+            except TypeError:
+                self.LOGGER.error("'dataset_parameters' section of metadata is incorrectly dumped.")
+            else:
+                if isinstance(json_loads_result, list):
+                    normalized_attributes['dataset_parameters'] = [
+                        get_cf_or_wkv_standard_name(dataset_param)
+                        for dataset_param in json_loads_result
+                    ]
+                else:
+                    self.LOGGER.error(
+                        "'dataset_parameters' section of metadata is not a json-dumped python list")
```

Comment:
Same question here, the ingester will continue executing with nothing in `dataset_parameters`.
Is it the desired behavior?

Reply:
done same as above